docs/apereo-incubation.md

```diff
@@ -40,8 +42,7 @@ Most major contributors have ICLAs on file.
 
 Next actions:
 
-+ Audit for CLA completeness of contributors
-+ Document CLA requirements in `CONTRIBUTING.md`
++ Follow up with past Contributors to secure CLAs
```

Comment:
Chance? Sure. 

Cf. [thoughts about partially adopting cla-assistant.io](https://github.com/UW-Madison-DoIT/angularjs-portal/blob/master/docs/apereo-incubation.md#413-contributor-license-agreements). On skim looks like `CLAHub` and `cla-assistant.io` are similar. I think one hangup is [concern that click-through ICLA agreement is not sufficient](https://groups.google.com/a/apereo.org/d/msg/projects/elNXbx5F-dE/Tu2xEqYhAgAJ). So there might have to be a layer of indirection where a Contributor *doesn't* agree to the ICLA via `cla-assistant.io`, rather they confirm that they've jumped through the real Apereo CLA-signing hoops, and do *that* confirmation via `cla-assistant.io`. So the end result of nice marking on the Pull Request still happens, but there's some wrinkles in the implementation and particularly the opportunity for Contributors to incorrectly affirm their having jumped through hoops they haven't fully jumped through.

Personally, [I'd love to see Apereo consider navigating the available policy tradeoffs differently](https://groups.google.com/a/apereo.org/d/msg/licensing-discuss/c1puG3RKZcA/Tonk3d23AQAJ).

If you'd like to help iterate Apereo licensing policy recommendations, please do prevail upon the uPortal project, say, to [nominate you to the re-forming Licensing Group](https://groups.google.com/a/apereo.org/d/msg/open/KnDFtuVo19U/lg4j0HsEBgAJ).